src/main/java/ui/components/FilterTextField.java

```diff
@@ -48,13 +53,15 @@ public FilterTextField(String initialText) {
     private void setup() {
         setPrefColumnCount(30);
         validationSupport.registerValidator(this, (c, newValue) -> {
-            boolean wasError = false;
+            boolean isValidFilter = true;
```

Comment:
Previously, `setTextFieldBackground` required a param `isValidFilter` and was called in `setup`. 

I thought that `setTextFieldBackground (isValidFilter)` was more reader friendly than `setTextFieldBackground(!wasError) so I changed it. 

But since the setting of the style of the background no longer require an argument. I think I will revert the variable back to `wasError`.